docs/operations/storageclasses.md

```diff
@@ -122,3 +122,44 @@ parameters:
   csi.storage.k8s.io/secret-name: storageos-api
   csi.storage.k8s.io/secret-namespace: storageos
 ```
+
+### Topology volume Storage Class
+
+StorageClass that utilizes kubernetes node topology zones and a list of
+preferred and required zones to decide where its deployments should live.
+
+More on [k8s storage-classes](https://kubernetes.io/docs/concepts/storage/storage-classes/#allowed-topologies)
+and csi specs for [create volume](https://github.com/container-storage-interface/spec/blob/master/spec.md#createvolume)
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: storageos-rep-enc
+provisioner: csi.storageos.com
+allowVolumeExpansion: true
+parameters:
+  csi.storage.k8s.io/fstype: ext4
+  csi.storage.k8s.io/secret-name: storageos-api
+  csi.storage.k8s.io/secret-namespace: storageos
+allowedTopologies:
+- matchLabelExpressions:
+  - key: topology.kubernetes.io/zone
+    values:
+    - eu-west-2
+    - us-east-1
+```
+
+Currently only the label "topology.kubernetes.io/zone" is supported.
+
+This label gets updated every hour but if you need a faster detection to a node label
+changing, it's possible to set a custom interval in the deployment/storageos-api-manager
+like so:
+
+```yaml
+containers:
+  - args:
+    - --node-label-resync-interval=1m
+```
+
+Where any input valid for golang's [time.ParseDuration](https://pkg.go.dev/time#ParseDuration) is accepted.
```

Comment:
Is this something that can be set in the cluster CR? Otherwise it's pretty difficult for user's to change this (especially if the operator keeps reverting changes)